gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Dialog.java

```diff
@@ -128,8 +128,8 @@ public Table getContentTable () {
 		return contentTable;
 	}
 
-	public Table getButtonTable () {
-		return buttonTable;
+	public Table getButtonsTable () {
```

Comment:
Not sure. I think the old name was ok. Who says there is more than one button? :D Plus old code using the old name continues to work.